vue3-apollo-quasar/src/components/ProfilePageLayout/ProfilePageLayout.vue

```diff
@@ -99,15 +104,36 @@ function changeTab(val) {
   tab.value = val;
 }
 
+const FILTER_TYPE_QUERY = gql`
+  query FilterType {
+    filterType @client
+  }
+`;
+
+const { result: filterTypeData, loading: loadingFilterType } =
+  useQuery(FILTER_TYPE_QUERY);
+
+const { repos, loading } = getUserRepos(props.username, false);
+
+const filteredTypeRepoData = computed(() => {
+  let resp: any = [];
```

Comment:
`resp` seems vague as a variable name.
perhaps name it something more idiomatic: eg. `response`

Also, can we avoid using `any` as a type?